src/test/java/teammates/test/cases/webapi/BaseActionTest.java

```diff
@@ -433,12 +434,22 @@ protected void verifyCanAccess(String... params) {
         c.checkAccessControl();
     }
 
+    protected void verifyCanAccess(FeedbackResponseUpdateRequest updateRequest, String... params) {
```

Comment:
The type is too specific for `BaseActionTest`? (i.e. what if we want to add this method for `FeedbackResponseCreateRequest`, `FeedbackSessionCreateRequest` etc.)

Reply:
Updated to now take in a `BasicRequest` instead